<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>281</height>
   </rect>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
background-color = brgb(45, 48, 51)
}</string>
  </property>
  <widget class="QLCDNumber" name="time_now">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="finish_class">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe 明體 Std L</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>准时下课</string>
   </property>
  </widget>
  <widget class="QLabel" name="now_subject">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>数学</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>2012</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前时间</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="rest_time">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>2012</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>剩余时间</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前科目</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>2012</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="text">
    <string>下课时间</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="over_time">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="mode">
    <enum>QLCDNumber::Dec</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>不拖堂的您最美</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
